SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -628,12 +670,12 @@ void ActorSystem::HandleDeferredEntityDeletion(const DeferredRetire& Retire) con
 {
 	if (Retire.bNeedsTearOff)
 	{
-		NetDriver->Sender->SendActorTornOffUpdate(Retire.EntityId, Retire.ActorClassId);
+		NetDriver->ActorSystem->SendActorTornOffUpdate(Retire.EntityId, Retire.ActorClassId);
```

Comment:
Do you think we can do this since it's the same object?
```suggestion
		SendActorTornOffUpdate(Retire.EntityId, Retire.ActorClassId);
```

Reply:
yep! And made the function const, thanks